src/NuGet.Clients/NuGet.PackageManagement.UI/GlobalSuppressions.cs

```diff
@@ -149,3 +149,4 @@
 [assembly: SuppressMessage("Build", "CA1501:'TabItemButton' has an object hierarchy '10' levels deep within the defining module. If possible, eliminate base classes within the hierarchy to decrease its hierarchy level below '6': 'Button, ButtonBase, ContentControl, Control, FrameworkElement, UIElement, Visual, DependencyObject, DispatcherObject, Object'", Justification = "<Pending>", Scope = "type", Target = "~T:NuGet.PackageManagement.UI.TabItemButton")]
 [assembly: SuppressMessage("Build", "CA1501:'VsDialogWindow' has an object hierarchy '11' levels deep within the defining module. If possible, eliminate base classes within the hierarchy to decrease its hierarchy level below '6': 'DialogWindow, DialogWindowBase, Window, ContentControl, Control, FrameworkElement, UIElement, Visual, DependencyObject, DispatcherObject, Object'", Justification = "<Pending>", Scope = "type", Target = "~T:NuGet.PackageManagement.UI.VsDialogWindow")]
 [assembly: SuppressMessage("Build", "CA1821:Remove empty Finalizers", Justification = "<Pending>", Scope = "member", Target = "~M:NuGet.PackageManagement.UI.DataStreamFromComStream.Finalize")]
+[assembly: SuppressMessage("Usage", "CA1816:Dispose methods should call SuppressFinalize", Justification = "<Pending>", Scope = "member", Target = "~M:NuGet.PackageManagement.UI.PackageManagerToolWindowPane.Dispose(System.Boolean)")]
```

Comment:
The class/method is calling `SuppressFinalize`. I imagine this warning doesn't need to be suppressed.